src/rest-server/src/models/v2/group.js

```diff
@@ -313,41 +258,31 @@ const initGrouplistInCfg = async () => {
     };
     await createGroupIfNonExistent(defaultVCGroup.groupname, defaultVCGroup);
     logger.info('Create default group successfully.');
-    logger.info('Create non-admin group configured in configuration.');
+    logger.info('Create group configured in configuration.');
     for (const groupItem of authConfig.groupConfig.grouplist) {
       await createGroupIfNonExistent(groupItem.groupname, groupItem);
     }
-    logger.info('Create non-admin group successfully.');
+    logger.info('Create group successfully.');
   } catch (error) {
-    logger.error('Failed to create admin group configured in configuration.');
+    logger.error('Failed to create group configured in configuration.');
     // eslint-disable-next-line no-console
     console.log(error);
   }
 };
 
-const createAdminUser = async () => {
+const createDefaultAdminUser = async () => {
+  const userModel = require('@pai/models/v2/user');
```

Comment:
Requre in the top

Reply:
In currently implementation, there exists a circular dependencies: group model depends on user model (when deleting group), user model depends on group mode (when checking admin and vc). CommonJS can't handle it correctly. As a workaround, we need require model on demand, instead at top.

Reply:
To solve it, I think we could remove the group model dependency, move related logic like deleting user grouplist into controller.